manuscript/response_to_review.Rmd

```diff
@@ -0,0 +1,121 @@
+
+**In Figure 1, I feel that something is missing here about how other scientists are supposed to interact with the data and forecasts.  The manuscript makes it pretty clear that this is not just for internal use, but is it actually inviting other labs to add models and make predictions? How specifically would another lab contribute a model to the project? Would they fork the project on github, develop a new model and then do a pull request? Or would they create a separate prediction archive themselves? Maybe add an arrow in Fig 1. showing how new models can be plugged into the forecasting work flow.  Box 1 addresses some of this but I guess I want more details.**
+
+We have added description of how others can contribute to the Modeling and Forecasting section of the paper. We tried to also add this to the figure, but could not find a way to do so without making the figure overly crowded and complicated.
+
+**Some other miscellaneous questions:**
+**Can models be dropped from the ensemble?**
+
+This is not currently done at the moment, but is on the development roadmap.
+
+**What if old data needs to be corrected or updated? That could change everything, including the interpretation of the archived predictions. Is there a way to handle that?**
+
+This is a great question about a tricky problem in forecast archiving that we have encountered and are still trying to determine out the optimal way to address. One can update the previous forecast files and use the versioning to maintain a copy of the forecasts that contained the underlying error or leave them as they are an store supplementary data indicating that forecasts prior to the selection should not be used in later analyses. Since we aren't happy with any of the current solutions, and this is a fairly detailed point related to archiving, we think it is best to leave this for later discussion.
+
+**Minor comments:**
+   
+**Lines 63 to 83:  Could this section include some examples that demonstrate the value of iterative near-term forecasting from other fields? Could it be argued that iterative near term forecasting has led to progress in other fields? Say meteorology and political science?**
+
+We have now mentioned the established benefits for iterative near term forecasting in meteorology and added several citations for readers interested in exploring this further. 
+
+**Figure 1: Another point on figure one.  What about model improvement? It seems like there should be a feedback loop somewhere showing how models get refined, added, dropped etc.**
+
+We tried to add a visual representation of this to Figure 1, but could not make it work without making the figure overly crowded. We have added some additional discussion of this "Modeling and Forecasting" section of the paper.
```

Comment:
discussion of this "Modeling and Forecasting" -> discussion of this to the "Modeling and Forecasting"